static/index.js

```diff
@@ -91,4 +91,50 @@ document.addEventListener("DOMContentLoaded", event => {
     }
   });
 
+  var darkSwitch = document.querySelector('input[name=darkMode]');
+  const prefersDarkScheme = window.matchMedia("(prefers-color-scheme: dark)");
```

Comment:
nit: we can make `prefersDarkScheme` a boolean.
```suggestion
  const prefersDarkScheme = window.matchMedia("(prefers-color-scheme: dark)").matches;
```